### ISSUE TITLE
Idea for ska version

### ISSUE THREAD

**Issue by taldcroft** [sot/skare3#23]
Following discussion in #19, one possibility is to have each meta package provide a version file.  Then the `ska_version` script would effectively paste all those together, giving a possibly long but unique version description of the environment.  E.g.
```
$SKA/versions/
    ska3-flight
    ska3-core
    ska3-pinned
    ska3-perl
```
Then `ska_version` might give something like:
```
flight-2018.09.22:core-2018.08.01:pinned-2018.08.01:perl-2018.08.01
```
If you decide to uninstall `ska3-flight` then it looks like `core-2018.08.01:pinned-2018.08.01:perl-2018.08.01`.

**Comment by jeanconn** [comment]
ska_version could also be a script that queries the versions of those packages from conda, but version files are probably simpler.

**Comment by taldcroft** [comment]
Per discussion, implement a ska_version script that gets package names and versions from conda list (JSON file) and looks for any package name starting with `ska3-`.  So this requires the convention of having versioned metapackaged named thusly (fine).

Output looks like below (alphabetized):
```
core-2018.09.22:flight-2018.08.01:pinned-2018.08.01:perl-2018.08.01
```

**Comment by jeanconn** [comment]
Closed by #37
